### PR TITLE
chore(android): remove unneeded warnings

### PR DESCRIPTION
**Pull request by jquick-axway**
**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28441

**Summary:**
- Remove below warning when calling blocking streaming API. We do not plan on removing this API in the future.
`Synchronous invocation of write will cause performance issues under the main thread. This will no longer be supported in SDK 10.0.0. Please invoke with a final callback function to receive the result.`
- Removed below image loading warning that needless spams the log. Calling code typically falls-back to the correct image path for cases where it has to guess the correct path. It's up to the caller to log an error or fire an error event.
`Could not open stream to get bitmap`

### CODE REVIEW

**Review by garymathews — APPROVED** [review]
CR: **PASS**

**Comment by jquick-axway** [comment]
@garymathews, do you think I should remove our 10.0.0 deprecation code comments below. Our API docs do not document the above exception properties as deprecated.
https://github.com/appcelerator/titanium_mobile/blob/master/android/titanium/src/java/org/appcelerator/titanium/TiExceptionHandler.java#L57
https://github.com/appcelerator/titanium_mobile/blob/master/android/titanium/src/java/org/appcelerator/titanium/TiExceptionHandler.java#L81

**Comment by garymathews** [comment]
@jquick-axway I think we should make them deprecated in 10.1.0 since we forgot about them in 9.0.0

File: apidoc/Titanium/App/App.yml [hunk 1]
```diff
@@ -309,22 +309,22 @@ events:
         type: String
         platforms: [android]
       - name: lineOffset
-        summary: The offset on the line where the error occurred.
+        summary: The offset on the line where the error occurred. (Deprecated since 9.0.0. Use `column` instead.)
```
[the remark this answers is not in the document]
Comment (jquick-axway) [reply apidoc/Titanium/App/App.yml:312]: The yml "deprecated" attribute is ignored for "event properties" in our old and new doc websites. Only the "summary" is used, which is why I'm duplicating this info there.
https://docs.appcelerator.com/platform/latest/#!/api/Titanium.App-event-uncaughtException
https://titaniumsdk.com/api/titanium/app.html#uncaughtexception

**Comment by lokeshchdhry** [comment]
FR Passed.

**Pull request merged**